requirements/deployment.in

```diff
@@ -1,4 +1,6 @@
-# These packages are installed from PyPI.
+django==3.2.23
+wagtail==4.2.4
+
```

Comment:
Should we consider merging these into the main list for simplicity? I get that these are the two biggies but we also have things like elasticsearch which are also pretty big...